plugins/session-tmux/internal/session/testdata/faketmux/main.go

```
@@ -73,6 +86,109 @@ func main() {
+func recordSession(socket, name string) {
+	if socket == "" || name == "" {
+		return
+	}
+
+	f, err := os.OpenFile(sessionsFile(socket), os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0o600)
+	if err != nil {
+		return
+	}
+	defer f.Close() //nolint:errcheck // best-effort record
+
+	fmt.Fprintln(f, name)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)